core/src/main/scala/org/apache/spark/status/AppStatusStore.scala

```diff
@@ -148,11 +148,20 @@ private[spark] class AppStatusStore(
     // cheaper for disk stores (avoids deserialization).
     val count = {
       Utils.tryWithResource(
-        store.view(classOf[TaskDataWrapper])
-          .parent(stageKey)
-          .index(TaskIndexNames.EXEC_RUN_TIME)
-          .first(0L)
-          .closeableIterator()
+        if (store.isInstanceOf[LevelDB]) {
```

Comment:
Could you invert the check so you don't need to import `LevelDB`? Just to avoid importing more implementation details of the kvstore module into this class...

Reply:
Done. Thanks